docs/docusaurus/src/components/NavbarItems/GithubNavbarItem/styles.module.css

```diff
@@ -6,6 +6,12 @@
     padding-right: 2rem;
 }
 
+@media (max-width: 996px) {
```

Comment:
Let's add these values into a css variable. is `996px` representative of a breakpoint? e.g. `small-screen`, `medium-screen`, `large-screen`, etc.

Reply:
I tried to create a --mobile-breakpoint css variable but css variables [don't work in media queries](https://stackoverflow.com/questions/40722882/css-native-variables-not-working-in-media-queries) (I didn't know that) 
The workaround that people suggest is media query the root statement
What you can do however is @media query your :root statement!
```
:root {
     /* desktop vars */
}

@media screen and (max-width: 479px) {
    :root {
        /* mobile vars */
    }
}
```
But I didn't find that solution appealing, especially since this github badge css is separated in it's own component css file. 
I agree that 996px SHOULD be a variable since its used in multiple places, but I had an impossible time trying to find a viable soluton, do you have any idea how I can achieve this?